api/openapi/my_api.yaml

```diff
@@ -225,263 +246,194 @@ components:
+      type: object
+      required:
+        - txid
+        - inputs
+        - cards
+      properties:
+        piid:
+          description: "A string, unique to this deployment, used to identify this plugin in realtime"
+          type: string
+          example: "pdspi-aminoglycoside-nomogram"
+        title:
+          description: "An optional title for the guidance plugin"
+          type: string
+          example: "Aminoglycoside dosing guidance"
+        txid:
+          type: string
+          description: "The transaction id sent in the GuidanceInputs. If none were sent, a unique id will be created and sent back here."
+          example: "38-1"
+        inputs:
+          type: object
+          items:
+            $ref: '#/components/schemas/Settings'
+        return:
+          type: object
+          items:
+            $ref: '#/components/schemas/Settings'
+        cards:
           type: array
           items:
-            type: string
-            description: "An array of container names upon which the plugin in question depends. For example, 'txscience/pds-fhir-example:0.2.0'. Mapper plugins must indicate a dependency on one or more FHIR plugins, but dependencies are discouraged otherwise in order to maintain the decoupling and reuse of plugins. Every plugin of type 'm' and 'mD' must have at least one 'f' or 'fD' plugin listed in pluginDependencies, otherwise return 500."
+            $ref: '#/components/schemas/Card'
+        specs:
+          type: array
+          items:
+            $ref: '#/components/schemas/Spec'
+        timeout:
+          type: integer
+          description: "Number of milliseconds until time out. Factory system default should be 120000 (2 minutes) for RUO applictions, and only 500ms for clinical."
+        fhir_piid:
+          type: string
+          example: "pdspi-fhir-example"
+          description: "Uniquely identifies the enabled and installed plugin to use for mapper"
+        mapper_piid:
+          type: string
+          example: "pdspi-mapper-example"
+          description: "Uniquely identifies the enabled and installed plugin to use for fhir"
+    Settings:
+      type: object
+      properties:
         pluginSelectors:
-          description: "Maps the piid to a set of selector values. Optionally call the /selectors endpoint first to get the full list of available selectors and their legal values. For example, two selectors might be 1. drug type and 2. indication. If the client selects values for the selector that matches all piid selector values, then this plugin can provide guidance. Potentially, a decision tree could be implemented here by prefacing each selectorarray item with logic, but for this API version we will instead always AND all the selector array elements. For example, 'if guidance is requried for drug _x_ AND indication _y_ then use plugin _z_'."
+          description: "Maps the piid to a set of selector values."
```

Comment:
This is really important documentation that needed to be conserved somewhere. I just pasted it into the Guidance schema of the PDS plugin, but it would be worthwhile maybe re-working this doc so it can be included everywhere? Perhaps this doc belongs on the Selector spec instead.

Reply:
@krobasky What exactly needs to be done here?

Reply:
don't delete taht documentation